org/urls.py

```diff
@@ -4,5 +4,6 @@
 
 urlpatterns = [
     path('', OrgView.as_view()),
+    path('<int:org_id>/',EditOrg,name='edit_org'),
     path('<int:org_id>/volunteer/',AddVolunteer,name='add_volunteer'),
 ]
```

Comment:
Haven't you performed linting? I guess the required package has been added to the repo and you just need to run it.
Even maybe switch it on, it will autorun at save(s). If there is any issue regarding linting, or it isn't working in editor, let's discuss that on slack.